project/CMakeLists.txt

```diff
@@ -1,5 +1,12 @@
-cmake_minimum_required(VERSION 3.8.0)
-cmake_policy(SET CMP0114 OLD)
+cmake_minimum_required(VERSION 3.12.0)
```

Comment:
should be 3.13 simdjson need 3.13

Reply:
Good catch. Actually the version of simdjson in third-party requires 3.14

I'll change it